media-libs/libffado/libffado-2.3.0.ebuild

```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 )
+[[ "${PV}" = "9999" ]] && inherit subversion
```

Comment:
Can you move this into the `if [ "${PV}" = "9999" ]` block on line 14?
It doesn't really matter where `subversion` is inherited here and I'd prefer to keep the if/logic blocks to a minimum.